src/Windows/Avalonia.Win32.Automation/Interop/Automation/UiaCoreProviderApi.cs

```diff
@@ -63,29 +63,57 @@ internal enum UiaEventId
         Changes
     };
 
-    internal static class UiaCoreProviderApi
+    internal static partial class UiaCoreProviderApi
     {
         public const int UIA_E_ELEMENTNOTENABLED = unchecked((int)0x80040200);
 
+#if NET7_0_OR_GREATER
```

Comment:
NET **_7_**?

Reply:
I use these compile time conditions with target versions that support specific feature.
In this context, LibraryImport was added in NET7_0_OR_GREATER, and it can be used independently from the new/old com interop.